vault/core.go

```diff
@@ -1284,6 +1298,11 @@ func NewCore(conf *CoreConfig) (*Core, error) {
 		return nil, err
 	}
 
+	c.limiterRegistry = conf.LimiterRegistry
```

Comment:
Your ENT PR also has `c.limiterRegistry.Logger = conf.Logger.Named("limits")` after this line. Was this omitted here on purpose?

Reply:
Yeah, this should be set earlier in `createCore`, so I dropped it here.